gemini/package-lock.json

```diff
@@ -3911,6 +3905,13 @@
         "connect-logger": "0.0.1",
         "lodash": "^4.11.1",
         "minimist": "1.2.0"
+      },
+      "dependencies": {
+        "minimist": {
+          "version": "1.2.0",
```

Comment:
This appears to be the wrong version here.

Reply:
Isn't this still the wrong version? Shouldn't it be greater than 1.2.0?

Reply:
`lite-server` requires this exact version.  

So to auto-update it we have to update `lite-server` version.  That is a dependency of gemini.

We are on the latest patch `7.4.1` the next version is `7.5.0` - I may try testing it tomorrow if it will work for us? 

Updating it by hand may be overwritten the next time someone updates some of the main packages. 

I will try to update as much as I can the packages but will take a little time (for testing).